app/errors.py

```diff
@@ -0,0 +1,14 @@
+class VaccineError(Exception):
```

Comment:
Consider adding a descriptive error message to your `VaccineError` class. For example, `class VaccineError(Exception):
    def __init__(self, message="An error occurred related to the vaccine."):
        super().__init__(message)`